src/Components/Card.tsx

```diff
@@ -87,15 +87,19 @@ const SaveButtonContainer = styled.div`
   top: 1rem;
 `
 
-export const Card = ({
-  id,
-  title,
-  shortDescription,
-  image,
-  costValue,
-  costQualifier,
-  age,
-}: ServicePreview): JSX.Element => {
+// Name the function Card so it shows up in DevTools
```

Comment:
I'm not sure this is making a difference for me - did you want to set the forwardRef.displayName for something?

Reply:
I checked; it shows up in the 'performance' tab when you take a profile, which is what I wanted.